docs/reference/recommended_practices.md

```diff
@@ -32,10 +32,84 @@
 - Should not refer to views in the `mozdata` project which are duplicates of views in another project
   (commonly `moz-fx-data-shared-prod`). Refer to the original view instead.
 - Should escape identifiers that match keywords, even if they aren't [reserved keywords]
+- Queries are interpreted as [Jinja](https://jinja.palletsprojects.com/en/3.1.x/) templates, so it is possible to use Jinja statements and expressions
 
 [spark-bigquery-connector]: https://github.com/GoogleCloudPlatform/spark-bigquery-connector/issues/5
 [reserved keywords]: https://cloud.google.com/bigquery/docs/reference/standard-sql/lexical#reserved-keywords
 
+### Querying Metrics
+
+- Queries, views and UDFs can reference metrics and data sources that have been defined in [metric-hub](https://mozilla.github.io/metric-hub/)
+  - To reference metrics use `{{ metrics.calculate() }}`:
+
+```sql
+SELECT
+  *
+FROM 
+  {{ metrics.calculate(
+    metrics=['days_of_use', 'active_hours'],
+    platform='firefox_desktop',
+    group_by={'sample_id': 'sample_id', 'channel': 'application.channel'},
+    where='submission_date = "2023-01-01"'
+  ) }}
+
+-- this translates to
+SELECT
+  *
+FROM
+  (
+  WITH clients_daily AS (
+    SELECT
+      client_id AS client_id,
+      submission_date AS submission_date,
+      COALESCE(SUM(active_hours_sum), 0) AS active_hours,
+      COUNT(submission_date) AS days_of_use,  
+    FROM
+      mozdata.telemetry.clients_daily
+    GROUP BY    
+      client_id,
+      submission_date
+  )
+  SELECT
+    clients_daily.client_id,
+    clients_daily.submission_date,
+    active_hours,
+    days_of_use,
+  FROM
+    clients_daily
+  )
+```
+    - `metrics`: unique to reference metric definition, all metric definitions are aggregations (e.g. SUM, AVG, ...)
```

Comment:
```suggestion
    - `metrics`: unique reference(s) to metric definition, all [metric definitions](https://mozilla.github.io/metric-hub/metrics/firefox_desktop/) are aggregations (e.g. SUM, AVG, ...)
```

Reply:
Not sure if that's the best URL because it goes to a specific platform (firefox_desktop), but link to the top level `/metrics/` doesn't work.